tests/integration/test_orders_cli.py

```diff
@@ -343,58 +341,76 @@ def test_cli_orders_create_basic_item_type_invalid(invoke):
         '--id', '4500474_2133707_2021-05-20_2419',
         '--bundle', 'analytic',
         '--item-type', 'invalid'
-        ])
+        ])  # yapf: disable
     assert result.exception
     assert 'Error: Invalid value: item_type' in result.output
 
 
 def test_cli_orders_create_id_empty(invoke):
     result = invoke([
         'create',
-        '--name', 'test',
-        '--id', '',
-        '--bundle', 'analytic',
-        '--item-type', 'invalid'
-        ])
+        '--name',
+        'test',
+        '--id',
+        '',
+        '--bundle',
+        'analytic',
+        '--item-type',
+        'invalid'
+    ])
```

Comment:
What do we do with cli options that are given as a list, where [OPTION NAME] and [OPTION VALUE] are separate entries? yapf wants to put them all on different lines. This seems to be a step back in readability, but I am meh to okay about letting yapf have it's way just to minimize any special cases we have in the code.

Reply:
@jreiberkyle we could switch to `'--item-type=invalid'` if we had to, that's a standard and click supports it.

Reply:
@sgillies ohh, great suggestion! I'll do that.